### PR TITLE
Tag LowRankModels v0.1.3 [https://github.com/madeleineudell/LowRankMo…

### PR DESCRIPTION
**Pull request by madeleineudell**
…dels.jl]

### CODE REVIEW

**Comment by aviks** [comment]
Quoting Travis: 

```
ERROR: LoadError: New tag 0.1.3 of package LowRankModels requires julia 0.5.0, 
but version 0.1.0 of LowRankModels requires julia 0.3.0-. Use a new minor package 
version when support for an old version of Julia is dropped. 
Re-tag the package as 0.2.0 using `Pkg.tag("LowRankModels", :minor)`.
```

**Comment by madeleineudell** [comment]
I'm having a problem with PkgDev creating a new minor version: locally, v0.2.0 already exists, but when I try to create v0.3.0, it accuses me of skipping over v0.2.0. So I can't increment the minor version. I've reset my local version of Metadata and still get the error. Where else might it be caching tag information?
```
julia> PkgDev.tag("LowRankModels",v"0.3.0")
ERROR: 0.3.0 skips over 0.2.0
 in check_new_version(::Array{VersionNumber,1}, ::VersionNumber) at ./version.jl:193
 in (::PkgDev.Entry.##37#45{String,Bool,String})(::Base.LibGit2.GitRepo) at /Users/madeleine/.julia/v0.5/PkgDev/src/entry.jl:259
 in with(::PkgDev.Entry.##37#45{String,Bool,String}, ::Base.LibGit2.GitRepo) at ./libgit2/types.jl:638
 in tag(::String, ::VersionNumber, ::Bool, ::String) at /Users/madeleine/.julia/v0.5/PkgDev/src/entry.jl:227
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},PkgDev.Entry.#tag,Tuple{String,VersionNumber,Bool}})() at ./pkg/dir.jl:31
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},PkgDev.Entry.#tag,Tuple{String,VersionNumber,Bool}}, ::String) at ./file.jl:59
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::String, ::Vararg{Any,N}) at ./pkg/dir.jl:31
 in tag(::String, ::VersionNumber) at /Users/madeleine/.julia/v0.5/PkgDev/src/PkgDev.jl:50

julia> PkgDev.tag("LowRankModels",v"0.2.0")
INFO: Tagging LowRankModels v0.2.0
ERROR: GitError(Code:EEXISTS, Class:Tag, Tag already exists)
 in macro expansion at ./libgit2/error.jl:99 [inlined]
 in (::Base.LibGit2.##74#76{Base.LibGit2.GitCommit,String,Bool,Base.LibGit2.GitRepo,String,Base.RefValue{Base.LibGit2.Oid}})(::Base.LibGit2.GitSignature) at ./libgit2/tag.jl:25
 in with(::Base.LibGit2.##74#76{Base.LibGit2.GitCommit,String,Bool,Base.LibGit2.GitRepo,String,Base.RefValue{Base.LibGit2.Oid}}, ::Base.LibGit2.GitSignature) at ./libgit2/types.jl:638
 in (::Base.LibGit2.##73#75{String,Bool,Base.LibGit2.Signature,Base.LibGit2.GitRepo,String,Base.RefValue{Base.LibGit2.Oid}})(::Base.LibGit2.GitCommit) at ./libgit2/tag.jl:24
 in with(::Base.LibGit2.##73#75{String,Bool,Base.LibGit2.Signature,Base.LibGit2.GitRepo,String,Base.RefValue{Base.LibGit2.Oid}}, ::Base.LibGit2.GitCommit) at ./libgit2/types.jl:638
 in #tag_create#72(::String, ::Bool, ::Base.LibGit2.Signature, ::Function, ::Base.LibGit2.GitRepo, ::String, ::String) at ./libgit2/tag.jl:22
 in (::Base.LibGit2.#kw##tag_create)(::Array{Any,1}, ::Base.LibGit2.#tag_create, ::Base.LibGit2.GitRepo, ::String, ::String) at ./<missing>:0
 in (::PkgDev.Entry.##37#45{String,Bool,String})(::Base.LibGit2.GitRepo) at /Users/madeleine/.julia/v0.5/PkgDev/src/entry.jl:263
 in with(::PkgDev.Entry.##37#45{String,Bool,String}, ::Base.LibGit2.GitRepo) at ./libgit2/types.jl:638
 in tag(::String, ::VersionNumber, ::Bool, ::String) at /Users/madeleine/.julia/v0.5/PkgDev/src/entry.jl:227
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},PkgDev.Entry.#tag,Tuple{String,VersionNumber,Bool}})() at ./pkg/dir.jl:31
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},PkgDev.Entry.#tag,Tuple{String,VersionNumber,Bool}}, ::String) at ./file.jl:59
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::String, ::Vararg{Any,N}) at ./pkg/dir.jl:31
 in tag(::String, ::VersionNumber) at /Users/madeleine/.julia/v0.5/PkgDev/src/PkgDev.jl:50
```

File: LowRankModels/versions/0.1.3/requires [hunk 1]
```diff
@@ -0,0 +1,9 @@
+julia 0.5
+DataFrames
+StatsBase
+ArrayViews
+Optim
+Roots
+NMF
+Compat
```
Comment (madeleineudell) [inline LowRankModels/versions/0.1.3/requires]: ok

**Comment by Keno** [comment]
> locally, v0.2.0 already exists

Delete 0.2.0 in both the repo and in METADATA?

**Comment by madeleineudell** [comment]
ok, added version numbers to require and updated new tag to v0.2.0. should be ready to merge.

**Comment by cstjean** [comment]
From the error log:

> ERROR: LoadError: LoadError: The following tags for LowRankModels have not been pushed to upstream repository: 0.2.0.
> To fix this, navigate to the package directory and run `git push --tags`.
> Afterward, close then reopen your METADATA pull request to restart the Travis CI build.

**Comment by kmsquire** [comment]
Bump.

**Comment by cstjean** [comment]
I just [pushed](https://github.com/madeleineudell/LowRankModels.jl/releases) the tag for 0.2.0. Can anyone please close and reopen this PR to restart Travis? This 0.2.0 tag is from August, so we'll likely tag 0.2.1 right after this PR is merged. Or should I delete 0.2.0 and retag it?

**Comment by tkelman** [comment]
Another tag would be a good idea, since the `test/runtests.jl` file doesn't include many of the other test files

**Pull request merged**